cmake/Buildwearables.cmake

```
@@ -23,3 +25,10 @@ ycm_ep_helper(wearables TYPE GIT
                       iDynTree
                       robometry
               CMAKE_ARGS ${WEARABLES_CMAKE_ARGS})
+
+if(ROBOTOLOGY_USES_PYTHON)
+  list(APPEND wearables_CONDA_DEPENDENCIES python)
+  list(APPEND wearables_CONDA_DEPENDENCIES pybind11)
+  # https://conda-forge.org/docs/maintainer/knowledge_base.html#pybind11-abi-constraints
+  list(APPEND bipedal-locomotion-framework_CONDA_DEPENDENCIES pybind11-abi)
```

```suggestion
  list(APPEND wearables_CONDA_DEPENDENCIES pybind11-abi)
```
I suppose this is a copy-paste typo.

---

Thanks for noticing!

---

Yes, thanks!